homeassistant/components/media_player/itunes.py

```diff
@@ -39,23 +52,23 @@ def __init__(self, host, port):
     def _base_url(self):
         """Return the base url for endpoints."""
         if self.port:
-            return self.host + ":" + str(self.port)
+            return 'http://{}:{}'.format(self.host, self.port)
```

Comment:
we should probably support an `ssl: true` configuration in case this is behind SSL. 

I personally previously did use `itunes-api` behind an nginx proxy, though I had HA configured to talking to it over localhost get those extra milliseconds, heh. But regardless, people might be behind HTTPS.